From 3c306b63f8377440c1ebdbf30264b342ff2d02d6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Thu, 15 May 2014 23:46:03 +0200
Subject: [PATCH] tests: pass PKCS12PATH to fix tests in out-of-source builds.

The set_pkcs12_cred used to default to looking for input files in a
subdirectory of the current working directory. When an out-of-source
build is performed, the files reside in a subdirectory of source
directory instead. Set PKCS12PATH to that directory in order to fix the
build.

Signed-off-by: Nikos Mavrogiannopoulos <nmav@gnutls.org>
---
 tests/Makefile.am |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index f155dfa..3fec620 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -114,6 +114,7 @@ TESTS_ENVIRONMENT =						\
 	PKCS12PASSWORD=foobar					\
 	PKCS12FILE_2=$(srcdir)/pkcs12-decode/pkcs12_2certs.p12	\
 	PKCS12PASSWORD_2=""					\
+	PKCS12PATH=$(srcdir)/pkcs12-decode/			\
 	EXEEXT=$(EXEEXT)					\
 	top_builddir="$(top_builddir)"				\
 	srcdir="$(srcdir)"
-- 
1.7.1

From b7f7b483a2c1e8da424bb3519e9365c489f1b248 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Thu, 15 May 2014 23:53:17 +0200
Subject: [PATCH] tests/slow: add -I flags necessary for out-of-source builds.

Signed-off-by: Nikos Mavrogiannopoulos <nmav@gnutls.org>
---
 tests/slow/Makefile.am |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/tests/slow/Makefile.am b/tests/slow/Makefile.am
index 0e4b382..d65b9d5 100644
--- a/tests/slow/Makefile.am
+++ b/tests/slow/Makefile.am
@@ -20,7 +20,9 @@
 AM_CFLAGS = $(WARN_CFLAGS) $(WERROR_CFLAGS)
 AM_CPPFLAGS = -I$(top_srcdir)/lib/includes		\
 	-I$(top_builddir)/lib/includes \
-	-I$(top_srcdir)/tests/
+	-I$(top_srcdir)/tests/ \
+	-I$(top_srcdir)/lib \
+	-I$(top_srcdir)/gl
 
 AM_LDFLAGS = -no-install
 LDADD = ../libutils.la \
-- 
1.7.1

